services/horizon/internal/db2/core/transaction.go

```diff
@@ -51,9 +51,10 @@ func (tx *Transaction) Fee(whiteListData map[string]string) int32 {
 
 func (tx *Transaction) isTxWhitelisted(whiteListData map[string]string) bool {
 	// First check if the source account is a Whitelist account.
-	if _, found := whiteListData[tx.Envelope.Tx.SourceAccount.Address()]; found {
-		return true
-	}
+	// TODO This check will be valid when this Jira task will be solved : https://kin.atlassian.net/browse/BC-814
```

Comment:
the link here is private and nobody outside of kin ecosystem can read it. i think it's better to explain the issue here in length